README.md

```
@@ -645,24 +659,27 @@ enum class LogLevel {
-SDK size is calculated as application size increases when _BlinkID_ SDK is added, with all its dependencies included.
+| ABI         | Download size | Install size |
+| ----------- | ------------- | ------------ |
+| armeabi-v7a | 3.51 MB       | 4.68 MB      |
+| arm64-v8a   | 3.57 MB       | 5.39 MB      |
+
+
+SDK size is calculated as application size increases when *BlinkID* SDK is added, with all its dependencies included.
+
+##  API documentation
+
+You can find the *BlinkID* SDK **KDoc** documentation [here](https://microblink.github.io/blinkid-android/index.html).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)